src/components/paymentMethodSelection/utils.ts

```diff
@@ -0,0 +1,47 @@
+import { formatPaymentMethodDetails } from '~/core/formats/formatPaymentMethodDetails'
+import { TranslateFunc } from '~/hooks/core/useInternationalization'
+import { PaymentMethodItem } from '~/hooks/customer/usePaymentMethodsList'
+
+const buildFooterParts = (
+  paymentProviderType?: string | null,
+  paymentProviderCode?: string | null,
+): string[] => {
+  const parts: string[] = []
+
+  if (paymentProviderType) {
+    parts.push(paymentProviderType)
+  }
+
+  if (paymentProviderType && paymentProviderCode) {
+    parts.push(' • ')
+  }
+
+  if (paymentProviderCode) {
+    parts.push(paymentProviderCode)
+  }
+
+  return parts
```

Comment:
Suggestion
You could do something like this:

```suggestion
  return [paymentProviderType, paymentProviderCode].filter(Boolean).join(' • ')
```

Reply:
great catch Alex. refactored!